frontend/src/book-read/components/read-drawer.component.jsx

```diff
@@ -42,9 +92,49 @@ export default function ReadDrawer() {
                 <ListItemIcon>
                   <ListAltIcon />
                 </ListItemIcon>
-                <ListItemText primary={'Browse books'} />
+                <ListItemText primary={"Browse books"} />
               </ListItemButton>
             </ListItem>
+            {!isSuccess ? (
+              <ListItem
+                component={Link}
+                sx={ListItemStyle}
+                to={`/${READ_CONFIG.routes.group}/${bookId}/review/new`}
+              >
+                <ListItemButton>
+                  <ListItemIcon>
+                    <ListAltIcon />
+                  </ListItemIcon>
+                  <ListItemText primary={"Review book"} />
```

Comment:
Please, choose appropriate Material Icons for the new items of this list here and below.